src/Mages.Core/Runtime/Functions/StandardFunctions.cs

```diff
@@ -686,6 +686,11 @@ public static class StandardFunctions
         (args.Length > 0 ? If.Is<Double>(args, SimpleRandom.CreateVector) : null) ??
         SimpleRandom.GetNumber());
 
+    /// <summary>
+    /// Contains the random integer function.
+    /// </summary>
+    public static readonly Function Randi = new(args => SimpleRandom.GetInteger((int)args[0]));
```

Comment:
As we don't know that `args[0]` is a double / convertable to an integer (or if we have at least 1 argument) I propose:

```cs
new(args => Curry.MinOne(Randi, args) ?? If.Is<Double>(args, x => SimpleRandom.GetInteger((int)x) ?? false)
```

Reply:
Just a newbie comment, but why are we checking for Double in the `If.Is<Double>` when we're actually looking for an integer?

Reply:
´There is no integer in Mages.

See the documentation: https://github.com/FlorianRappl/Mages/blob/devel/doc/types.md

Reply:
Oh ok, gotcha 🫡